android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/auth/InMemoryDAO.java

```diff
@@ -0,0 +1,77 @@
+package org.sagebionetworks.bridge.android.manager.auth;
+
+import android.content.Context;
+import android.support.annotation.AnyThread;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import org.sagebionetworks.bridge.rest.model.UserSessionInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Created by jyliu on 1/20/2017.
+ */
+@AnyThread
+class InMemoryDAO implements AuthenticationManager.DAO {
```

Comment:
Yes, this is a placeholder. There will be another implementation. Michael is working on how to store credentials and I'll integrate with that.

(This is another reason I created BridgeDataProvider2, AuthManager doesn't have sufficient features to replace the old code)